bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,334 @@
+This binding shall bring the benefits of over-the-network control of the [Clementine-Player] to OpenHab!
+
+It allows control of playback, position and volume and allows display of various title information:
+
+* Artist
+* Album
+* Title
+* current position within track
+* Album cover
+
+
+[→ useful Links][links]
+
+## Discovery
+
+This thing does (currently) not provide auto-discovery.
+You will have to manually configure clementine to allow remote connections.
+And you will have to manually configure the Thing instances to talk to your player:
+
+## Thing Configuration
+
+Configuring the connection to Clementine is quite easy:
+
+Normally you only need to set the hostname your Clementine player is running at.
+If you set a password in the Clementine configuration, you will have to provide it to the configuration, too.
+
+### `sample` Thing Configuration
+
+| Name     | Type    | Description                                                       | Default | Required | Advanced |
+|----------|---------|-------------------------------------------------------------------|---------|----------|----------|
+| hostname | text    | Hostname or IP address of the device                              | N/A     | yes      | no       |
+| password | text    | Password to access the device                                     | N/A     | no       | no       |
+| port     | integer | Port at which Clementine is listening for remote control commands | 5500    | yes      | no       |
+
+## Channels
+
+The binding provides the following channels:
+
+| Channel          | Type          | Read/Write  | Description                                  |
+|------------------|---------------|-------------|----------------------------------------------|
+| album            | String        | read-only   | Album the currently playing song belongs to  |
+| artist           | String        | read-only   | Artist of the currently playing title        |
+| cover            | Image         | read-only   | Cover of the current album                   |
+| playback-control | String        | read/write  | Common control of playback and position      |
+| position         | Number:Time   | read-only   | Current position within the playing track    |
+| state            | String        | read-only   | Current state of Clementine player           |
+| title            | String        | read-only   | Name of the currently playing track          |
+| track            | String        | read-only   | Number of the title within the current album |
+| volume-control   | system.volume | read/write  | Playback volume                              |
+
+
+
+## Full Example
+
+### Thing
+
+```
+UID: clementineremote:clementine:0590462b5b
+label: Clementine
+thingTypeUID: clementineremote:clementine
+configuration:
+  port: 5500
+  hostname: 192.168.1.6
+```
+
+### Widget
+
+based on https://community.openhab.org/t/universal-remote-widget/118423
+
+```
```

Comment:
Please add yaml to the code block. I also think it is mandatory to add full examples of textual configuration for *.things and *.items files.

Reply:
Fixed.

Reply:
yaml has been added, but the textual config is missing and i'm not sure if the yaml code (quite large) is of added value.

Reply:
Pardon me, I don't think I understand what you mean with textual config here.

Reply:
https://www.openhab.org/docs/configuration/#textual-vs-graphical-configuration

Reply:
So, I understand the concepts. But still, I have no clue how I can get grip on the textual config of my thing. I thought, this is what is presented under the "Code" tab when I edit my thing.

Reply:
That is YAML, with textual config usually the *.things and *.items files that are on disk and edited from any text editor or vs code. Examples can be found in most addon readme.md files. An example: https://www.openhab.org/addons/bindings/plugwiseha/#plugwiseha-things

Reply:
Took me a while to figure this out. https://www.openhab.org/docs/configuration/things.html helped a lot.

Fixed.